licenses/Alembic

```diff
@@ -0,0 +1,63 @@
+TM & © 2009-2015 Lucasfilm Entertainment Company Ltd. or Lucasfilm Ltd.
+All rights reserved.
```

Comment:
I am not sure what line 1-2 do here, but at line 4 BSD-3 clause starts: https://opensource.org/licenses/BSD-3-Clause
How to proceed with the Attachment?